faq.md

```diff
@@ -5,64 +5,150 @@ layout: faq
+
+- the server can support a room full of people all doing the same analysis at the same time.
+- all tools and reference datasets needed in the tutorial are locally installed.  Some tutorials include scripts that install all needed tools and data on an existing server.
+- all participants will be able to create/use accounts on the system.
+
+TODO: Training Infrastructure as a Service on useGalaxy.eu
+
+Finally, you can also run your training on cloud-based infrastructures.  Galaxy has been available on Amazon Web Services for years, and is also available on many national research infrastructures such as Jetstream (United States), GenAP (Canada), GVL (Australia), CLIMB (United Kingdom), and so on.
+
+## What are the best practices for teaching with Galaxy?
 
-- Parse the website to find interesting tutorials to teach
-- Find a Galaxy instance to use for the workshop
-    - Check the annotated Galaxy instance
-    - Training Infrastructure as a Service on useGalaxy.eu
-    - Docker image
-    - Deployment of the tools, data, etc on an exisiting instance using our provided scripts
 - Have a look at our [Good practices slides]()
 
-# How can I make sure a tutorial is running on a Galaxy instance?
 
-- Badge?
-- Installations
+## How do I get help?
+
+The support channel for instructors is the same as for individual learners.  We suggest you start by posting a question to the Galaxy Training Network [Gitter chat](https://gitter.im/Galaxy-Training-Network/Lobby).  Anyone can view the discussion, but you'll need to login (using your GitHub or Twitter account) to add to the discussion.
+
+If you have questions about Galaxy in general (that are not training-centric) then there are [several support options](https://galaxyproject.org/support/).
+
+
+# Contributing
+
+First off, thanks for your interest in contributing to the Galaxy training materials!
+
+Individual learners and instructors can make these training more effective by contributing back to them. You can report mistakes and errors, create more content, etc. Whatever is your background, there is a way to contribute.  See the [contributing page]({{ site.github_repository }}/CONTRIBUTING.md)) for how.
 
-# How can I contribute?
+TODO: @tnabtaf suggests moving rest of this section to CONTRIBUTING.md
 
-First off, thanks for taking the time to contribute!
+: via the GitHub website, via command-line or even without dealing with GitHub.
 
-You can report mistakes or errors, create more contents, etc. Whatever is your background, there is probably a way to do it: via the GitHub website, via command-line or even without dealing with GitHub.
 
 You can check our [tutorials](http://galaxyproject.github.io/training-material/topics/contributing) for more details.
 
 We will address your issues and/or assess your change proposal as promptly as we can, and help you become a member of our community.
 
+## How can I give feedback?
+
+- tutorial level
+- contributions
```

Comment:
We should put there the link to the contributor feedback when we will have it. @tnabtaf do you think we can have/send it before GCC? It could be nice then to advertise it there